closure/templates/BUILD

```diff
@@ -33,6 +33,7 @@ closure_js_library(
     name = "soy_jssrc",
     srcs = ["@com_google_template_soy_jssrc"],
     lenient = True,
+    suppress = ["missingSourcesWarnings"],
```

Comment:
could you clarify what this is?

Reply:
This is not something we can safely ignore. Could you try upgrade soy to `36c7400e51c71c0af2b8b569e613a15f05deb994`?
If doesn't work, then you can try `309707311325543e936fd14df24dd026c3051737`.

Reply:
Ok jspb dependency introduced then the directionality is removed so there wouldn't be a working version.

Can try to patch in [this diff](https://github.com/google/closure-templates/commit/309707311325543e936fd14df24dd026c3051737) in http_archive to see if it is enough?

Reply:
You are not simply suppressing a warning; it is invalid call that accessing a property that doesn't exist. The patch is fixing that.

Regardless we will remove soy support after the release. I just wanted to make sure we don't make a release with soy completely broken.